internal/controllers/clusterextension_controller.go

```diff
@@ -223,8 +231,11 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 
 	post := &postrenderer{
 		labels: map[string]string{
-			util.CoreOwnerKindKey: ocv1alpha1.ClusterExtensionKind,
-			util.CoreOwnerNameKey: ext.GetName(),
+			util.CoreOwnerKindKey:          ocv1alpha1.ClusterExtensionKind,
+			util.CoreOwnerNameKey:          ext.GetName(),
+			util.ResolvedbundleName:        bundle.Name,
+			util.ResolvedbundlePackageName: bundle.Package,
+			util.ResolvedbundleVersion:     bundleVersion.String(),
```

Comment:
No catalogID here? Can we drop that constant?

Reply:
https://github.com/operator-framework/operator-controller/pull/765